Azure/sonic-utilities
# [intfutil] Display RJ45 port and portchannel speed in 'M' instead of 'G' when it's <= 1000M

#### What I did
To follow the convention of 1000Base-T Gigabit Ethernet, display the RJ45 port speed in 'M' if it's <= 1000M, instead of 'G', e.g:
10M, 100M, 1000M; 
For the portchannel which is composed of multi RJ45 ports, will also follow the same convention.

#### How I did it
Verify the port type from the TRANSCEIVER_INFO table, RJ45 port or portchannel will display the port speed in the unit 'M' instead of 'G'

#### How to verify it
Run "show interface status" command on platforms with RJ45 port to check the port speed is correct.

#### Previous command output (if the output of a command-line utility has changed)
if the speed is less than 1G then it can not be displayed correctly.
```
admin@sonic:/# show in st
  Interface            Lanes    Speed    MTU    FEC    Alias    Vlan    Oper    Admin             Type    Asym PFC
-----------  ---------------  -------  -----  -----  -------  ------------  ------  -------  ---------------  ----------
  Ethernet0                0        G   9100    N/A     etp1        routed      up       up             RJ45         N/A
  Ethernet1                4        G   9100    N/A     etp2        routed      up       up             RJ45         N/A
  Ethernet2                8       1G   9100    N/A     etp3        routed      up       up             RJ45         N/A
  Ethernet3               12       1G   9100    N/A     etp4        routed      up       up             RJ45         N/A
  Ethernet4               16       1G   9100    N/A     etp5  PortChannel1      up       up             RJ45         N/A
  Ethernet5               20       1G   9100    N/A     etp6        routed      up       up             RJ45         N/A
  Ethernet48 192,193,194,195     100G   9100    N/A    etp49        routed      up       up  QSFP28 or later         N/A
  Ethernet52 196,197,198,199     100G   9100    N/A    etp50        routed      up       up  QSFP28 or later         N/A
  Ethernet56 200,201,202,203     100G   9100    N/A    etp51        routed      up       up  QSFP28 or later         N/A
  Ethernet60 204,205,206,207     100G   9100    N/A    etp52        routed      up       up  QSFP28 or later         N/A
PortChannel1             N/A       1G   9100    N/A      N/A        routed    down       up              N/A         N/A

```

#### New command output (if the output of a command-line utility has changed)
admin@sonic:/# show in st
```
   Interface            Lanes    Speed    MTU    FEC    Alias          Vlan    Oper    Admin             Type    Asym PFC
------------  ---------------  -------  -----  -----  -------  ------------  ------  -------  ---------------  ----------
   Ethernet0                0     100M   9100    N/A     etp1        routed      up       up             RJ45         N/A
   Ethernet1                4     100M   9100    N/A     etp2        routed      up       up             RJ45         N/A
   Ethernet2                8    1000M   9100    N/A     etp3        routed      up       up             RJ45         N/A
   Ethernet3               12    1000M   9100    N/A     etp4        routed      up       up             RJ45         N/A
   Ethernet4               16    1000M   9100    N/A     etp5  PortChannel1      up       up             RJ45         N/A
   Ethernet5               20    1000M   9100    N/A     etp6        routed      up       up             RJ45         N/A
  Ethernet48  192,193,194,195     100G   9100    N/A    etp49        routed      up       up  QSFP28 or later         N/A
  Ethernet52  196,197,198,199     100G   9100    N/A    etp50        routed      up       up  QSFP28 or later         N/A
  Ethernet56  200,201,202,203     100G   9100    N/A    etp51        routed      up       up  QSFP28 or later         N/A
  Ethernet60  204,205,206,207     100G   9100    N/A    etp52        routed      up       up  QSFP28 or later         N/A
PortChannel1              N/A    1000M   9100    N/A      N/A        routed    down       up              N/A         N/A
```
